app/components/card-expense.tsx

```diff
@@ -34,21 +34,23 @@ export function CardExpense({
         <ul>
           {expenses.map((expense) => (
             <li className="my-2" key={expense.transaction_code}>
-              <div className="flex justify-between">
-                <h3 className="text-lg font-medium">{expense.description}</h3>
-                <h3 className="text-lg font-medium">
-                  {new Intl.NumberFormat("id-ID", {
-                    style: "currency",
-                    currency: "IDR",
-                  }).format(expense.idr_amount.toNumber())}
-                </h3>
-              </div>
+              <a href={`/expenses/123`}>
```

Comment:
Can you at least put a TODO for someone else to change the `123` into something else?